### PR TITLE
Examples : set sampler to always sample

### PR DESCRIPTION
**Pull request by mayurkale22**

### CODE REVIEW

**Comment by mayurkale22** [comment]
#1387 @songy23 Please review, thanks @briantq for the report.

File: examples/src/main/java/io/opencensus/examples/Utils.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.examples;
+
+public class Utils {
```
Comment (songy23) [inline examples/src/main/java/io/opencensus/examples/Utils.java]: This class could be package-private.

File: examples/src/main/java/io/opencensus/examples/Utils.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  public static void sleep(int ms) {
```
Comment (songy23) [inline examples/src/main/java/io/opencensus/examples/Utils.java]: Same here.

**Comment by bogdandrutu** [comment]
One important caveat here:
* Usually people COPY-PASTE examples in their code and the "always sampler" can get to their production and cause large bills when they use a tracing service like Stackdriver.

We need to make sure this is not the case, so probably a big WARNING is good to be added.

File: examples/src/main/java/io/opencensus/examples/trace/MultiSpansScopedTracing.java [hunk 3]
```diff
@@ -55,9 +59,27 @@ private static void doWork() {
...
    */
   public static void main(String[] args) {
+
+    // WARNING: Be careful before you set sampler value to always sample, especially in
+    // production environment. Trace data is often very large in size and is expensive to
+    // collect. This is why rather than collecting traces for every request(i.e. alwaysSample),
+    // downsampling is prefered.
+    //
+    // By default, OpenCensus provides a probabilistic sampler that will trace once in every
+    // 10,000 requests. If you prefer to use probabilistic sampler, you might not see trace data
```
Comment (bogdandrutu) [inline examples/src/main/java/io/opencensus/examples/trace/MultiSpansScopedTracing.java]: s/". If you prefer to use probabilistic sampler"/", that's why if default probabilistic sampler is used here"

and everywhere where you have this comment.

**Pull request merged**